src/exporters/xcmTransfers.ts

```diff
@@ -0,0 +1,82 @@
+import * as PromClient from "prom-client"
+import { getParachainName, decimals } from '../index';
+import { logger} from '../logger';
+import { Exporter } from './IExporter';
+import { ApiPromise } from "@polkadot/api";
+import { Header } from "@polkadot/types/interfaces";
+
+class XCMTransfersExporter implements Exporter {
+	palletIdentifier: any;
+	xcmTransfers: PromClient.Gauge< "chain" | "tochain">
+
+	constructor(registry: PromClient.Registry) {
+		this.palletIdentifier = "system";
```

Comment:
this pallet is called `xcmPallet`. The point of this is that somewhere in the main file, we do `(if api.query[palletIdentifier] !== undefined) {}` do stuff. 

Now you are basically saying if the system pallet exists in a chain, then scape the metrics for it, which is wrong.